email_template_qweb/views/mail_template.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="email_template_form" model="ir.ui.view">
+        <field name="model">mail.template</field>
```

Comment:
perhaps would be great to add a name to these views, just as a little improvement. WDYT? 😄